### DISCUSSION TITLE
incorrect require-bundle in slf4j-simple to slf4j-api since 2.0.6

### DISCUSSION THREAD

**Discussion by Nutstree** [qos-ch/slf4j#338] [General]
Since 2.0.6 the symbolic names are changed in SLF4J which introduced an inconsistency

slf4j-simple MANIFEST contains this
`Require-Bundle: slf4j.api`

while at the same time slf4j-api MANIFEST contains this
`Bundle-SymbolicName: org.slf4j.api`

**Comment by ceki** [comment]
Please associate PRs and bug reports to a [Jira ](https://jira.qos.ch) issue.

**Comment by Nutstree** [comment]
bug report created in jira: https://jira.qos.ch/browse/SLF4J-583

**Comment by selundqma** [comment]
I also experience this when developing my Eclipse RCP application (using Eclipse 2022-12). Manually updating the slf4j-simple-2.0.6.jar/META-INF/manifest.mf to include "Require-Bundle: org.slf4j.api" instead of "Require-Bundle: slf4j.api" solves it. Hoping for a fix in 2.0.7 :).

**Comment by ceki** [comment]
@Nutstree  The jira.qos.ch server crashed and had to be rebuilt from backups. This led to some data loss. If you would like to
follow issues on the Jira, please register with jira.qos.ch a second time. My apologies for the inconvenience.
